app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -1,62 +1,11 @@
-.historic-people-index {
-  .clear-person {
-    clear: both;
+.historic-people-index__section-row-header {
+  @include media(tablet) {
```

Comment:
There's a media-query mixin from the Design System that's more future proof, so I think it should be used here instead of `media()`.

This suggestions assumes that the `min-width` is tablet, rather than the `max-width`:

```suggestion
  @include govuk-media-query($from: "tablet") {
```

Reply:
I keep reaching for the wrong media query snippet 🙈 - will update, thanks for spotting!